main_functional_test.go

```diff
@@ -202,8 +202,12 @@ var _ = Describe("Functional", func() {
 					writeWant := fmt.Sprintf("Successfully wrote message to exchange '%s'", exchangeName)
 					Expect(writeGot).To(ContainSubstring(writeWant))
 
-					// Now try and read from the SQS queue
-					readCmd := exec.Command(
+					ctx, cancel := context.WithTimeout(context.Background(), 20*time.Second)
```

Comment:
Added these to see which test is being flaky and causing the pipeline to fail. Can't seem to reproduce it via CI